mitodl/bootcamp-ecommerce
# Fixed UI to show only payable klasses

#### What are the relevant tickets?
Fixes #63

#### What's this PR do?
Fixes the UI to show only payable klasses. A user will only be able to select/pay for klasses that are set to `is_user_eligible_to_pay=true`

#### How should this be manually tested?
Unfortunately we don't have a test bootcamp server or a straightforward way to return some fake results for that API. You can use [this patch](https://gist.github.com/gsidebo/699c66e0a780e6fa269838db1508a0c0) to get the API to return every `Bootcamp`/`Klass` in your database that doesn't already have an associated `Line`. Those `Klasses` will all be considered payable. 

Make sure to test that a result set with 1 payable `Klass` will not show a dropdown, and a result set with multiple payable `Klasses` will show all of them in the dropdown

#### Where should the reviewer start?
`static/js/containers/PaymentPage.js`

#### Any background context you want to provide?
- A lot of this diff is related to the variable name changes from `KlassIndex` to `KlassKey` et. al.
- I'm not sure why `static/js/containers/Payment.js` was still hanging around. That file was renamed to `PaymentPage.js` a few PRs ago. I got rid of it as part of this PR
